packages/venia-ui/package.json

```diff
@@ -60,6 +60,7 @@
     "@babel/preset-env": "~7.3.4",
     "@pmmmwh/react-refresh-webpack-plugin": "0.4.1",
     "@storybook/react": "~6.1.14",
+    "@testing-library/react-hooks": "~5.0.3",
```

Comment:
Wouldn't it make sense to have this in main repo instead of `venia-ui`? `peregrine` might use it one day?

Reply:
It is already in peregrine 😄  I'm still waiting on my flow chart on how to determine where deps go, but happy to take your lead on this. I should add to the root package and remove from these two?

Reply:
Now I am confused 😓 . If we add it to the main repo and don't add in `venia-ui` and `peregrine`, for someone using these packages individually, they won't have the package in their `node_modules` which will cause problems while running tests.

So I guess, we have to add it in `venia-ui` and `peregrine`'s dev dependencies list along with the main repo so for us it will be a single installation and the individual packages will pickup from higher `node_modules` and for someone using these packages individually, it will from the `node_modules` of the package itself.

Does that make sense? 😅